chainer/functions/math/tensordot.py

```diff
@@ -0,0 +1,175 @@
+import collections
+import numpy
+import six
+
+from chainer.backends import cuda
+from chainer import function_node
+from chainer import utils
+from chainer.utils import type_check
+
+
+def _tensordot(a, b, a_axes, b_axes, c_axes=None):
+
```

Comment:
Quite a nitpick but I think we can remove this empty line!

Reply:
All right, will remote that empty line :)